dspace/config/spring/cris/cris-relationpreference.xml

```diff
@@ -96,7 +96,16 @@
 		<property name="relationName" value="crisevents.publications" />
 		<property name="relationClass" value="org.dspace.content.Item" />
 		<property name="query">
-			<value>dc.relation.conference_authority:{0}</value>
+			<value>dc.relation.conference_authority:{0} OR '{'!join from=cris-id to=dc.relation.conference_authority fromIndex=search'}'crisevents.eventsparent_conference_hsu_authority:{0}</value>
```

Comment:
this is the query to retrieve all the publication that are linked to the current event or to a children event (the join query). Please note that in the join query the cris-id must match the _authority of the metadata used in the publication to link the cris object (the event in this case) and the query `crisevents.eventsparent_conference_hsu_authority` is the one used to identify children of the current event